### DISCUSSION TITLE
no index.php file

### DISCUSSION THREAD

**Discussion by Kmaduwantha** [SYMMKA/web#89] [Q&A] [answered]
there is no index file in the web app how to host this?

**Comment by Manu1ND** [comment, marked as answer]
You can keep the project in your web server (like www folder or htdocs for XAMPP) and navigate to "http://localhost/sih2020/web/admin" in your browser.
The admin folder has the "home.php" file so it will redirect automatically to that page.

**Reply by Kmaduwantha** [reply]
thanks brother
